packages/nouns-contracts/contracts/governance/fork/NounsDAOV3Fork.sol

```diff
@@ -216,6 +220,11 @@ library NounsDAOV3Fork {
         return ds.forkEscrow.numTokensInEscrow();
     }
 
+    function checkNounIdIsAllowedToFork(NounsDAOStorageV3.StorageV3 storage ds, uint256 tokenId) internal view {
+        uint256 auctionedNounId = INounsAuctionHouseV2(ds.nouns.minter()).auction().nounId;
+        if (tokenId < auctionedNounId - ds.nounAgeRequiredToFork) revert NounIdNotOldEnough();
```

Comment:
gas: worth reading `ds.nounAgeRequiredToFork` and `auctionedNounId` once for all the tokens